js/graviex.js

```diff
@@ -0,0 +1,991 @@
+        //         "side": "sell"
+        //     },
+        //     {
+        //         "id": 14473615,
+        //         "at": 1643670349,
+        //         "price": "0.00000059",
+        //         "volume": "515.0136",
+        //         "funds": "0.000303858024",
+        //         "market": "giobtc",
+        //         "created_at": "2022-02-01T02:05:49+03:00",
+        //         "side": "buy"
+        //     },
+        // ],
+        return this.parseTrades (response, market, since, limit);
+    }
+
+    parseOHLCV (ohlcv, market = undefined) {
+        return [
+            this.safeInteger (ohlcv, 0),
+            this.safeNumber (ohlcv, 1),
+            this.safeNumber (ohlcv, 2),
+            this.safeNumber (ohlcv, 3),
+            this.safeNumber (ohlcv, 4),
+            this.safeNumber (ohlcv, 5),
+        ];
+    }
+
+    async fetchOHLCV (symbol, timeframe = '1m', since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const defaultLimit = 50;
+        const maxLimit = 1500;
+        limit = (limit === undefined) ? defaultLimit : Math.min (limit, maxLimit);
+        const request = {
+            'market': market['id'],
+            'limit': limit,
+        };
+        const response = await this.publicGetK (this.extend (request));
+        // [
+        //     [
+        //         1643719320,
+        //         6.5e-7,
+        //         6.5e-7,
+        //         6.5e-7,
+        //         6.5e-7,
+        //         0,
+        //     ],
+        //     [
+        //         1643719380,
+        //         6.5e-7,
+        //         6.5e-7,
+        //         6.5e-7,
+        //         6.5e-7,
+        //         0,
+        //     ],
+        // ],
+        return this.parseOHLCVs (response, market, timeframe, since, limit);
+    }
+
+    async fetchOrderBook (symbol, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'market': market['id'],
+        };
+        if (limit !== undefined) {
+            request['limit'] = limit; // default 100,
+        }
+        const response = await this.publicGetDepth (this.extend (request, params));
+        // {
+        //   "timestamp": 1643721617,
+        //   "asks": [
+        //     [
+        //       "0.00000253",
+        //       "120.0"
+        //     ],
+        //     [
+        //       "0.0000025",
+        //       "2424.0288"
+        //     ],
+        //     [
+        //       "0.00000249",
+        //       "77.577"
+        //     ],
+        //     [
+        //       "0.00000248",
+        //       "12.5"
+        //     ],
+        // },
+        const timestamp = this.safeInteger (response, 'timestamp');
+        const orderbook = this.parseOrderBook (response, symbol, timestamp);
+        orderbook['nonce'] = undefined;
```

Comment:
this line is uneccesary

Reply:
Okay. Fixed in the latest commit https://github.com/ccxt/ccxt/pull/11697/commits/162b106bdb42084917d6187a3a60fb6e0f55e7d1